backend/compact-connect/lambdas/purchases/config.py

```diff
@@ -0,0 +1,44 @@
+import json
+import logging
+import os
+from functools import cached_property
+
+import boto3
+from aws_lambda_powertools.logging import Logger
+
+logging.basicConfig()
+logger = Logger()
+logger.setLevel(logging.DEBUG if os.environ.get('DEBUG', 'false').lower() == 'true' else logging.INFO)
+
+
+class _Config:
+    presigned_post_ttl_seconds = 3600
```

Comment:
Where is this used, and for what purpose?

Reply:
good catch - copy/paste from other directory. I'll remove it.